src/spreadinterp.cpp

```diff
@@ -1815,8 +1806,8 @@ int indexSort(std::vector<BIGINT> &sort_indices, UBIGINT N1, UBIGINT N2, UBIGINT
     did_sort = 1;
   } else {
 #pragma omp parallel for num_threads(maxnthr) schedule(static, 1000000)
-    for (BIGINT i = 0; i < M; i++) // here omp helps xeon, hinders i7
-      sort_indices[i] = i;         // the identity permutation
+    for (BIGINT i = 0; i < BIGINT(M); i++) // here omp helps xeon, hinders i7
```

Comment:
Here I need to use a signed loop counter, because  MSVC insists on signed loop counters in OpenMP constructs. This has been relaxed in the OpenMP standard for quite some time, but Microsoft apparently doesn't care.